deployments/arbitrum/usdc.n/migrations/1686953660_configurate_and_ens.ts

```diff
@@ -0,0 +1,278 @@
+    const officialMarketsJSON = JSON.parse(await ENSResolver.text(subdomainHash, ENSTextRecordKey));
+    if (officialMarketsJSON[arbitrumChainId]) {
+      officialMarketsJSON[arbitrumChainId].push(newMarketObject);
+    } else {
+      officialMarketsJSON[arbitrumChainId] = [newMarketObject];
+    }
+
+    const mainnetActions = [
+      // 1. Set Comet configuration and deployAndUpgradeTo new Comet on Arbitrum.
+      {
+        contract: arbitrumInbox,
+        signature: 'createRetryableTicket(address,uint256,uint256,address,address,uint256,uint256,bytes)',
+        args: [
+          bridgeReceiver.address,                           // address to,
+          0,                                                // uint256 l2CallValue,
+          createRetryableTicketGasParams.maxSubmissionCost, // uint256 maxSubmissionCost,
+          refundAddress,                                    // address excessFeeRefundAddress,
+          refundAddress,                                    // address callValueRefundAddress,
+          createRetryableTicketGasParams.gasLimit,          // uint256 gasLimit,
+          createRetryableTicketGasParams.maxFeePerGas,      // uint256 maxFeePerGas,
+          l2ProposalData,                                   // bytes calldata data
+        ],
+        value: createRetryableTicketGasParams.deposit
+      },
+      // 2. Approve the COMP gateway to take Timelock's COMP for bridging
+      {
+        contract: COMP,
+        signature: 'approve(address,uint256)',
+        args: [compGatewayAddress, COMPAmountToBridge]
+      },
+      // 3. Bridge COMP from mainnet to Arbitrum rewards
+      {
+        contract: arbitrumL1GatewayRouter,
+        signature: 'outboundTransferCustomRefund(address,address,address,uint256,uint256,uint256,bytes)',
+        args: [
+          COMP.address,                             // address _token,
+          refundAddress,                            // address _refundTo,
+          rewards.address,                          // address _to,
+          COMPAmountToBridge,                       // uint256 _amount,
+          compGasParams.gasLimit,                   // uint256 _maxGas,
+          compGasParams.maxFeePerGas,               // uint256 _gasPriceBid,
+          utils.defaultAbiCoder.encode(
+            ['uint256', 'bytes'],
+            [compGasParams.maxSubmissionCost, '0x']
+          )                                         // bytes calldata _data
+        ],
+        value: compGasParams.deposit
+      },
+      // 4. Update the list of official markets
+      {
+        target: ENSResolverAddress,
+        signature: 'setText(bytes32,string,string)',
+        calldata: ethers.utils.defaultAbiCoder.encode(
+          ['bytes32', 'string', 'string'],
+          [subdomainHash, ENSTextRecordKey, JSON.stringify(officialMarketsJSON)]
+        )
+      },
+      // 5. Displace v2 USDT COMP rewards
```

Comment:
USDT v2 COMP speeds are already set to 0, yeah? We shouldn't need to set it to 0 again

Reply:
I see.  Yeah will remove it.  

I was a bit confused about why there is v2 rewards in arbitrum migration script. 😅